packages/babel-preset-env/src/normalize-options.js

```diff
@@ -250,15 +263,25 @@ export default function normalizeOptions(opts: Options) {
 
   const corejs = normalizeCoreJSOption(opts.corejs, useBuiltIns);
 
-  const include = expandIncludesAndExcludes(
+  let include = validateIncludeExcludeOption(
```

Comment:
Seems cleaner to just move the `validateIncludeExcludeOption` call inside `expandIncludesAndExcludes`?